src/picongpu/include/plugins/kernel/CopySpecies.kernel

```diff
@@ -33,8 +33,104 @@ namespace picongpu
 
 using namespace PMacc;
 
+/** copy particle to single frame
+ *
+ * copy particle data which are stored in linked list and superCell hierarchy
+ * to a single frame.
+ */
+struct CopySpeciesToGlobal
```

Comment:
But I think we can add these specialities to the description and call it `ConcatListOfFrames` still.